### PR TITLE
added wal-g example to docs

### PR DESCRIPTION
**Pull request by guruguruguru**

### CODE REVIEW

**Comment by sgotti** [comment]
@guruguruguru Thanks for your PR. Can you please squash in a single commit?

**Comment by guruguruguru** [comment]
@sgotti done

File: doc/pitr.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 ## Point in time recovery
 
 Stolon can do a point a time recovery starting from an existing backup.
-This example shows how to do point in time recovery with stolon using [wal-e](https://github.com/wal-e/wal-e)
+[This](pitr_wal-g.md) example shows how to do point in time recovery with stolon using [wal-g](https://github.com/wal-g/wal-g).
+[This](pitr_wal-e.md) example shows how to do point in time recovery with stolon using [wal-e](https://github.com/wal-e/wal-e)
```
Comment (sgotti) [inline doc/pitr.md]: You should make this a list or will render all in the same line

Comment (guruguruguru) [reply doc/pitr.md]: done

**Comment by sgotti** [comment]
@guruguruguru Thanks! Merging.

**Pull request merged**